corehq/apps/smsbillables/models.py

```diff
@@ -343,6 +345,13 @@ def create(cls, message_log, multipart_count=1):
         return billable
 
     @classmethod
+    def _get_multipart_count(cls, backend_api_id, backend_instance, backend_message_id, multipart_count):
+        if backend_api_id == SQLTwilioBackend.get_api_id():
+            return int(get_twilio_message(backend_instance, backend_message_id).num_segments)
```

Comment:
would be nice to only make one call to get the message from twilio per each billable request